packages/mobile/package.json

```diff
@@ -72,13 +72,13 @@
     "date-fns": "^2.12.0",
     "dot-prop-immutable": "^1.4.0",
     "ethereumjs-util": "^6.2.0",
+    "fp-ts": "2.1.1",
     "fuzzysort": "^1.1.4",
     "google-libphonenumber": "^3.2.4",
     "graphql": "^14.1.1",
     "hermes-engine": "0.4.2-rc1",
     "i18next": "^19.4.3",
     "io-ts": "2.0.1",
-    "fp-ts": "2.1.1",
```

Comment:
🙏 👨‍🎨  it's the little things, thank you for alphabetizing

Reply:
Don't thank me. Thank automation 🤖